README.md

```diff
@@ -1,5 +1,32 @@
 # Constitution
 
-The governing rules of Hackers at Cambridge.
+The governing rules of "Hackers at Cambridge", hereafter (and in the rest of the repository) referred to as the "Society".
 
-[Read the constitution here](constitution.md).
+[Read the constitution here](constitution.md). You can also find a web version
+[on our website](https://hackersatcambridge.com/constitution).
+
+# This Repository
+
+As laid out in the [constitution](constitution.md#society-repository), this repository is used to list all rules and guidelines of the Society that are not defined or contained in the constitution. This repository exists for the benefit of the executive committee, but is free to read for the general public for those who may be interested.
+
+As for why we went with this setup, we will be posting more information about this soon!
```

Comment:
Is it useful to have this here? Seems like the sort of thing that will likely fall through the cracks...